colors/mc2.txt

```diff
@@ -187,18 +187,28 @@ mcl_comparators:comparator_off_sub 113 101 97
 mcl_comparators:comparator_on_comp 115 101 97
 mcl_comparators:comparator_on_sub 116 101 97
 mcl_core:acacialeaves 141 136 76
+mcl_trees:acacialeaves 141 136 76

```

Comment:
the leaves follow these scheme:

mcl_trees:leaves_acacia
mcl_trees:leaves_oak
mcl_trees:leaves_dark_oak
mcl_trees:leaves_cherry_blossom